bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard1.5</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard1.5</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard1.5;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard1.5;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
How does this work? Is line 5 simply *overwriting* what line 4 did when the `$OS` is not `Windows_NT`?

Reply:
BUT... this means we HAVE to build the Nuget package on Windows right?

Because the Nuget package MUST have artifacts for `net452`